packages/terra-clinical-detail-view/src/terra-dev-site/test/clinical-detail-view/DetailView/DetailViewOverflowText.test.jsx

```diff
@@ -8,9 +8,9 @@ const accessoryElement = (<p> A sample accessory container </p>);
 
 const DetailViewOverflowText = () => (
   <DetailView
-    title="OverflowTextOverflowTextOverflowTextOverflowTextOverflowTextOverflowTextOverflowTextOverflowTextOverflowText"
-    secondaryTitles={['secondaryTitle1', 'secondaryTitle2']}
-    subtitles={['Subtitle 1', 'Subtitle 2']}
+    title={'OverflowText'.repeat(10)}
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat it seems this maybe not be compatible on IE? Do this render as expected in these browsers?

Reply:
They appear to be working

### IE10
<img width="1634" alt="screen shot 2018-06-19 at 4 35 04 pm" src="https://user-images.githubusercontent.com/17147005/41625544-c7af3f26-73de-11e8-8174-4753b8c62833.png">

### IE11
<img width="1633" alt="screen shot 2018-06-19 at 4 33 38 pm" src="https://user-images.githubusercontent.com/17147005/41625545-c7c2b2fe-73de-11e8-8e14-f6a09ce93b08.png">